stage7/01-adi-kernel-dts/00-run.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+
+LINUX_DIR="${1:-linux-adi}"
+DTFILE="$2"
+CROSS_COMPILE="$3"
+
+HOST=${HOST:-i686}
+DEFCONFIG_PI0=adi_bcmrpi_defconfig
+DEFCONFIG_PI3=adi_bcm2709_defconfig
+DEFCONFIG_PI4=adi_bcm2711_defconfig
+GCC_ARCH=arm-linux-gnueabi
+IMG_NAME="zImage"
+KERNEL_IMG_PI0=kernel.img
+KERNEL_IMG_PI3=kernel7.img
+KERNEL_IMG_PI4=kernel7l.img
+ARCH=arm
+
+[ -n "$NUM_JOBS" ] || NUM_JOBS=5
+
+LINARO_GCC_VERSION="5.5.0-2017.10"
```

Comment:
this GCC version is a bit old;
we could leave it like this for now;

the wiki scripts get a new GCC
https://github.com/analogdevicesinc/wiki-scripts/blob/master/linux/build_rpi_kernel_image.sh

an alternative is to wget the wiki scripts and use them;
that way the build updates as those scripts update

Reply:
i could do that. I would update the script to update the kernel for pi0 and pi4 also use a parameter for defconfig. I think this would be best. What are your thoughts?